packages/webapp/src/containers/Finances/index.js

```diff
@@ -355,63 +356,58 @@ class Finances extends Component {
     return (
       <div className={styles.financesContainer}>
         <h4>
-          <strong>FINANCES</strong>
+          <strong>{this.props.t('SALE.FINANCES.TITLE')}</strong>
         </h4>
         <hr/>
-        <h4><b>Action</b></h4>
+        <h4><b>{this.props.t('SALE.FINANCES.ACTION')}</b></h4>
         <div className={styles.buttonContainer}>
           <Button onClick={() => {
             history.push('/expense_categories')
-          }}>Add New Expense </Button>
+          }}>{this.props.t('SALE.FINANCES.ADD_NEW_EXPENSE')}</Button>
           <Button onClick={() => {
             history.push('add_sale')
-          }}>Add New Sale</Button>
+          }}>{this.props.t('SALE.FINANCES.ADD_NEW_SALE')}</Button>
         </div>
         <hr/>
         <DateRangeSelector changeDateMethod={this.changeDate}/>
```

Comment:
use i18n in `packages/webapp/src/components/Finances/DateRangeSelector/index.js`